src/data-type.js

```diff
@@ -93,7 +95,13 @@ for (const id in TYPE) {
       const alias = ref[i];
       if (!typeByName[alias]) {
         typeByName[alias] = type;
+
+        deprecate.property(typeByName, alias, 'The `' + alias + '` data type alias is deprecated, please use `' + type.name + '` instead.');
       }
     }
   }
 }
+
+['IntN', 'BitN', 'FloatN', 'MoneyN', 'DateTimeN'].forEach(function(name) {
+  deprecate.property(typeByName, name, 'The `' + name + '` data type is internal and will be removed.');
```

Comment:
DecimalN and NumericN should also be added for deprecation.

Reply:
> DecimalN and NumericN should also be added for deprecation.

Ah, yes, I missed those! 😅 I added them both.